### PR TITLE
Issue/581/add icon and splash screen for android and ios app

### LINKED ISSUE #581: Add icon and splash screen for Android and iOS app

### PR DESCRIPTION
**Pull request by dbetka**
close #581 

**Example `File filter` settings to review important changes:**

>

Not important files have extensions such as:
- grandle
- png
- properties
- xcf

___

**Done:**
- Add icons and splash screen to mobiles
- Add scripts for generating icons and splash screens for mobiles and docs about it
- Add docs about styles in Android app
- Set own theme for android app (status bar, splash screen)
- Set own settings for iOS status and navigation bar
- Add colors to sass variables for status and navigation bar (only iOS)
- Correct icons size
- Rename `resources` to `mobile-resources`
- Split icon file to Android and iOS ones
- Add messages on splash screen for server connection pending and error
- Add error server connection popup on splash screen
- Hide splash screen onMounted for loading-app component
- Change color of navigation bar on android and remove unnecessary ic_launcher icons

**Done - not related:**
- Ignore type errors for field validation - required to build production package
- Add SSL certificates for domains to Android app for connection with server

**Unable to do:**
- Test on real iOS device because I don't have an adapter to install app on iPhone
___

**Android package to for test on real device:**
https://www.dropbox.com/s/q5dlthbf34g79o5/HarcMap-2.0.1.apk?dl=0

___

**Size of icons on devices**

 

___

**Splash screen with messages and popups**

To reproduce change `testMode` on `true` in `client/src/components/organisms/loading-app.vue:57`.

### CODE REVIEW

File: client/android/app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="colorPrimary">#07632c</color>
+    <color name="colorPrimaryDark">#07632c</color>
+    <color name="colorAccent">#ffffff</color>
```
Comment (CleverDarkSolutions) [inline client/android/app/src/main/res/values/colors.xml:5]: Accent is a weird name imo, we can debate on this

**Review by CleverDarkSolutions — CHANGES_REQUESTED** [review]

File: client/android/app/src/main/res/values/colors.xml (hunk 1 above)
Comment (dbetka) [reply client/android/app/src/main/res/values/colors.xml:5]: I'll fix it

**Review by CleverDarkSolutions — CHANGES_REQUESTED** [review]

File: client/src/components/organisms/loading.vue [hunk 2]
```diff
@@ -25,6 +25,25 @@
...
+          >
+            {{ 'Sprawdź połączenie z internetem' }}
+          </div>
+        </transition>
+        <transition name="longer-fade">
+          <div
+            v-if="loadingCounter > 1 && loadingCounter <= 3"
+            class="a-text-status-loading"
+          >
+            {{ 'Oczekiwanie na połączenie z serwerem' }}
```
[the remark this answers is not in the document]
Comment (dbetka) [reply client/src/components/organisms/loading.vue:43]: Translations will be added

File: client/android/app/src/main/res/values/colors.xml (hunk 1 above)
Comment (dbetka) [reply client/android/app/src/main/res/values/colors.xml]: Done

Comment (dbetka) [reply client/android/app/src/main/res/values/colors.xml]: Done

File: client/src/components/organisms/loading.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (dbetka) [reply client/src/components/organisms/loading.vue]: Done

**Pull request merged**